integration-tests/pom.xml

```diff
@@ -53,6 +53,7 @@
         <module>elytron-resteasy</module>
         <module>elytron-undertow</module>
         <module>flyway</module>
+        <module>liquibase</module>
         <module>oidc</module>
         <module>oidc-code-flow</module>
         <module>keycloak-authorization</module>
```

Comment:
Note: we also need to add the native tests to the pipeline here: https://github.com/quarkusio/quarkus/blob/master/ci-templates/stages.yml#L187 .

Reply:
Done